config/pathdef.c.in

```diff
@@ -3,5 +3,6 @@
 #include "${PROJECT_SOURCE_DIR}/src/nvim/vim.h"
 char *default_vim_dir = "${CMAKE_INSTALL_FULL_DATAROOTDIR}/nvim";
 char *default_vimruntime_dir = "";
+char *default_lib_dir = "${CMAKE_INSTALL_FULL_LIBDIR}/nvim";
```

Comment:
```suggestion
// Separate from $VIMRUNTIME to avoid binaries in "PREFIX/share". 
char *default_lib_dir = "${CMAKE_INSTALL_FULL_LIBDIR}/nvim";
```

Reply:
Probably better in option.c. But more docs in general is needed, let's do it in a doc PR. (the point of this PR is to get something minimal to get started, we might need to revisit desicions during the 0.5 cycle)